package.json

```diff
@@ -46,6 +48,7 @@
     "blueimp-canvas-to-blob": "^3.5.0",
     "browser-encrypt-attachment": "^0.3.0",
     "browser-request": "^0.3.3",
+    "chokidar": "^1.6.1",
```

Comment:
should be in devDependencies; it's not needed by anyone trying to use react-sdk as a lib